lib/screens/home/explorer/explorer_bloc.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:async';
+
+import 'package:get_it/get_it.dart';
+import 'package:provenance_wallet/services/models/provenance_validator.dart';
+import 'package:provenance_wallet/util/strings.dart';
+
+class ExplorerBloc extends Disposable {
+  @override
+  FutureOr onDispose() {
+    // TODO: implement onDispose
+    throw UnimplementedError();
+  }
+}
+
+class StakingDetails {
+  StakingDetails({
+    required this.delegates,
+    required this.validators,
+    this.selectedType = Strings.dropDownDelegate,
+    this.selectedStatus = Strings.dropDownAllStatuses,
+    required this.address,
+  });
+
+// FIXME: delegates are a different type eventually
+  List<ProvenanceValidator> delegates;
```

Comment:
These properties should probably be `final` to avoid mutation and subsequent side-effects.